target_chains/ethereum/entropy_sdk/js/src/examples/EvmRandomDraw.ts

```diff
@@ -0,0 +1,126 @@
+import Web3 from "web3";
+import yargs from "yargs";
+import { hideBin } from "yargs/helpers";
+
+import HDWalletProvider from "@truffle/hdwallet-provider";
+import IEntropy from "@pythnetwork/entropy-sdk-solidity/abis/IEntropy.json";
+import { FortunaConnection } from "../index";
+
+const argv = yargs(hideBin(process.argv))
+  .option("network", {
+    description: "RPC for the network to relay on.",
+    type: "string",
+    required: true,
+  })
+  .option("contract", {
+    description: "Entropy contract address.",
+    type: "string",
+    required: true,
+  })
+  .option("mnemonic", {
+    description: "Mnemonic (private key) for sender",
+    type: "string",
+    required: true,
+  })
+  .option("fortuna-url", {
+    description: "URL for Fortuna. e.g: https://endpoint/example",
+    type: "string",
+    required: false,
+  })
+  .option("provider", {
+    description:
+      "The randomness provider to query" + " e.g: 0xf9c0172ba10dfa4d19088d...",
+    type: "string",
+    required: false,
+  })
+  .help()
+  .alias("help", "h")
+  .parserConfiguration({
+    "parse-numbers": false,
+  })
+  .parseSync();
+
+const network = argv.network;
+
+async function run() {
+  const walletProvider = new HDWalletProvider({
+    mnemonic: {
+      phrase: argv.mnemonic,
+    },
+    providerOrUrl: network,
+  });
+  console.log(`wallet address: ${walletProvider.getAddress(0)}`);
+
+  // @ts-ignore
+  const web3 = new Web3(walletProvider);
+
+  const userRandom = web3.utils.randomHex(32);
+  console.log(`userRandom: ${userRandom}`);
+  const commitment = web3.utils.keccak256(userRandom);
```

Comment:
users will need this code snippet to generate a random number and the corresponding commitment. However, this is tied to the web3.js library, which not everyone uses (some use ethers etc) so not sure people will want that dependency.